_amenities/tennis-information.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: default
+title: Tennis Information
+date: 2019-05-28 06:04
+author: Klahanie
+comments: true
+categories: [amenities]
+banner: /images/tennis.jpg
+show-title: true
+---
+
+<div class="alert alert-warning">
+<strong>Note</strong> Klahanie tennis courts are closed as a result of the COVID-19 pandemic
+</div>
+
+Klahanie will be offering youth and adult tennis lessons throughout the summer for those that are looking for a great way to stay active, learn a new sport (or improve, if you're already a pro!) and have fun while enjoying the outdoors.

```

Comment:
"for those who are looking"
The CMoS uses the Oxford/serial comma, which I also have a strong affinity for. If we want to adopt the Oxford comma for our future style guide, we'd add a comma here: "already a pro!), and"